plugins/modem-manager/fu-mm-device.c

```diff
@@ -125,6 +129,9 @@ validate_firmware_update_method(MMModemFirmwareUpdateMethod methods, GError **er
 	    MM_MODEM_FIRMWARE_UPDATE_METHOD_FIREHOSE,
 #if MM_CHECK_VERSION(1, 19, 1)
 	    MM_MODEM_FIRMWARE_UPDATE_METHOD_FIREHOSE | MM_MODEM_FIRMWARE_UPDATE_METHOD_SAHARA,
+#endif
+#if MM_CHECK_VERSION(1, 23, 0)
+		MM_MODEM_FIRMWARE_UPDATE_METHOD_CINTERION_FDL,
```

Comment:
this isn't new to your pull request, but your pull request prompts me to ask the question.  Since there is this check for `supported_combinations`, should we instead of failing probe give the user a better message about the situation in an update error?

Reply:
@hughsie ^

Reply:
I'm not sure update error is entirely expected -- i mean -- what's the user going to do with that information?

Reply:
File a bug with their distro to update the software?

Reply:
So I think that would be a PROBLEM, i.e. something like `FWUPD_DEVICE_PROBLEM_OUTDATED_SOFTWARE` -- so it gets set as an update error --but localized.